### DISCUSSION TITLE
Should screenshots/screen capture be disabled when password protection is on?

### DISCUSSION THREAD

**Discussion by naveensingh** [FossifyOrg/General-Discussion#141] [Polls]
The ability to disable screenshots or screen capture was requested by some people (emails). I'd like to know what other people think about this. Whatever option you choose, please also comment your reason below.

**Comment by tswistak** [comment]
I think there should be an option for that. Password protection doesn't mean you keep something secret in the app, that you wouldn't like to take a screenshot of. It may be just a protection, e.g. before kids messing in your notes.

**Reply by naveensingh** [reply]
I see your point, I changed my vote :)
